EssentialFeed/EssentialFeedTests/ImageComments/Presentation/ImageCommentsPresenterTests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Copyright © 2019 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentsPresenterTests: XCTestCase {
+	func test_title_isLocalized() {
+		XCTAssertEqual(ImageCommentsPresenter.title, localized("IMAGE_COMMENTS_TITLE"))
+	}
+
+	func test_map_createsViewModel() {
+		let now = Date()
+		let calendar = Calendar(identifier: .gregorian)
+		let locale = Locale(identifier: "en_US_POSIX")
+		let comments = [
+			ImageComment(id: UUID(), message: "message1", createdAt: now.adding(minutes: -5), username: "username1"),
+			ImageComment(id: UUID(), message: "message2", createdAt: now.adding(days: -1), username: "username2")
+		]
+		let viewModel = ImageCommentsPresenter.map(comments, currentDate: now, calendar: calendar, locale: locale)
```

Comment:
Missing line break to separate the test sections:

```suggestion
		]
		
		let viewModel = ImageCommentsPresenter.map(comments, currentDate: now, calendar: calendar, locale: locale)
```

Reply:
Thank you. I fixed it.